api/v1beta2/foundationdb_database_configuration.go

```diff
@@ -49,6 +49,9 @@ type DatabaseConfiguration struct {
 	// Regions defines the regions that the database can replicate in.
 	Regions []Region `json:"regions,omitempty"`
 
+	// ExcludedServers defines the list  of excluded servers form the database.
+	ExcludedServers []ExcludedServers `json:"excluded_servers,omitempty"`
```

Comment:
We have to ignore this slice when we compare the current and the desired configuration otherwise the operator will compare the currently excluded servers in this comparison.